pypose/lietensor/basics.py

```diff
@@ -40,6 +40,17 @@ def vec2skew(input:torch.Tensor) -> torch.Tensor:
                         torch.stack([ v[...,2],         O, -v[...,0]], dim=-1),
                         torch.stack([-v[...,1],  v[...,0],         O], dim=-1)], dim=-2)
 
+def scalar2skew(scalar:torch.Tensor) -> torch.Tensor:
```

Comment:
Is there a better name?

Reply:
extended vec2skew to include this case, there this was removed.